src/routes/users/index.ts

```diff
@@ -243,4 +244,21 @@ router.post<Express.EmptyParams, { token: string }, { role: Roles }, void, Requi
     })
 );
 
+type IntrospectParams = {
+    token: string;
+};
+
+router.get<IntrospectParams, UserInfo, void, void>(
+    '/introspect/:token',
```

Comment:
Lets make token a query parameter or passed in using the body.  Typically using the `/`'s indicate that there's some id of some collection or subcollection. ie `/townhalls/:townhallId`